src/main/java/org/sagebionetworks/web/server/servlet/oauth2/OAuth2AliasServlet.java

```diff
@@ -0,0 +1,72 @@
+package org.sagebionetworks.web.server.servlet.oauth2;
+
+import java.io.IOException;
+import java.net.URLEncoder;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.sagebionetworks.client.SynapseClient;
+import org.sagebionetworks.repo.model.LogEntry;
+import org.sagebionetworks.repo.model.oauth.OAuthProvider;
+import org.sagebionetworks.repo.model.oauth.OAuthValidationRequest;
+import org.sagebionetworks.repo.model.principal.PrincipalAlias;
+import org.sagebionetworks.util.SerializationUtils;
+import org.sagebionetworks.web.shared.WebConstants;
+
+public class OAuth2AliasServlet extends OAuth2SessionServlet {
```

Comment:
Change so that both OAuth2AliasServlet and OAuth2SessionServlet share the same parent (or are composed with the same object)